lib/greek_stemmer.rb

```diff
@@ -242,8 +242,14 @@ def stem(word)
       end
     end
 
+    # steb 6b
+    stem.scan(/^(.+?)(ΟΥΑ)$/ux) do |st, suffix|
```

Comment:
From rubular: `x ignore whitespace in regex` . Never payed much attention :smile: , it's used throughout.
The parentheses are needed in general, because the `suffix` block parameter will be `nil`, but in this case it's not used so we could omit it.